### DISCUSSION TITLE
Metro CLI tool improvement - auth/login -> auth_login_page [unintended]

### DISCUSSION THREAD

**Discussion by rakhaxor** [nylo-core/nylo#95] [Ideas] [answered]
Hello there! I just started using Nylo today. I am from Laravel background and obviously used the artisan CLI a lot. I was trying to create a page using `metro make:page auth/login, this in turn created auth_profile_page.dart which was not intended. Excuse me if this is something I am not aware of and is already there. I went through the whole documentation and the youtube videos but found nothing about this thing. Thank you!

**Comment by agordn52** [comment]
Hi @rakhaxor,

Thanks for the comment, hope you're enjoying the festive break!
I've just tried this on my machine and I'm experiencing the same. I'll make some changes now and create a new release.

I'll let you know when it's live 👍

**Reply by rakhaxor** [reply]
Thanks for the prompt response! I will be waiting for the changes, let me know if I could help in doing it (I am just a beginner in Flutter tho but experienced developer). I really appreciate your efforts for the community!

**Reply by agordn52** [reply]
Hi @rakhaxor,

Happy New Year! 

I've updated the package to support this now, you can try it [v5.17.0](https://pub.dev/packages/nylo_framework).

**Reply by rakhaxor** [reply]
Happy new year to you my friend! Thank you for the wish and doing the changes. I will try them tomorrow. I am wondering if I just need to run flutter pub upgrade or there’s some manual work I need to do to successfully upgrade to the latest Nylo version.

**Reply by agordn52** [reply, marked as answer]
No problem! 

<ins>Re. how to upgrade</ins>
You will just need to update the dependency version in your pubspec.yaml file.

~~nylo_framework: ^5.16.0~~
`nylo_framework: ^5.17.0`

Hope that helps

**Comment by rakhaxor** [comment]
Awesome! I will now mark this thread as resolved.   Thank you Anthony!
